documentation/latest/gsg/minikube.md

```diff
@@ -0,0 +1,147 @@
+```shell
+---> minikube status
+minikube
+type: Control Plane
+host: Running
+kubelet: Running
+apiserver: Running
+kubeconfig: Configured
+```
+
+
+Flotta operator has a few tools that helps to install flotta, for that, let's
+clone it:
+
+```shell
+git clone https://github.com/project-flotta/flotta-operator -b main --depth 1
+cd flotta-operator
+```
+
+First, we need to install Openshift-router, the ingress that it's supported now:
+
+```shell
+---> make install-router
+kubectl apply -f https://raw.githubusercontent.com/openshift/router/master/deploy/router_rbac.yaml
+clusterrole.rbac.authorization.k8s.io/openshift-ingress-router created
+clusterrolebinding.rbac.authorization.k8s.io/openshift-ingress-router created
+clusterrolebinding.rbac.authorization.k8s.io/openshift-ingress-router-auth-delegator created
+namespace/openshift-ingress created
+serviceaccount/ingress-router created
+kubectl apply -f https://raw.githubusercontent.com/openshift/router/master/deploy/route_crd.yaml
+customresourcedefinition.apiextensions.k8s.io/routes.route.openshift.io created
+kubectl apply -f https://raw.githubusercontent.com/openshift/router/master/deploy/router.yaml
+deployment.apps/ingress-router created
+```
+
+Now let's install Flotta on the cluster:
+
+```shell
+make deploy IMG=quay.io/project-flotta/flotta-operator:latest TARGET=k8s
+```
+
+A bunch of CRDs are now created, where the definitions can be found
+[here](../operations/crd.md):
+
+```
+---> kubectl  api-resources | grep flotta
+edgeconfigs                                    management.project-flotta.io/v1alpha1   true         EdgeConfig
+edgedevices                                    management.project-flotta.io/v1alpha1   true         EdgeDevice
+edgedevicesets                                 management.project-flotta.io/v1alpha1   true         EdgeDeviceSet
+edgedevicesignedrequest           edsr         management.project-flotta.io/v1alpha1   true         EdgeDeviceSignedRequest
+edgeworkloads                                  management.project-flotta.io/v1alpha1   true         EdgeWorkload
+playbookexecutions                             management.project-flotta.io/v1alpha1   true         PlaybookExecution
+--->
+```
+
+At the same time, in the flotta namespace an operator should be running:
+```
+---> kubectl -n flotta get pods
+NAME                                                  READY   STATUS    RESTARTS   AGE
+flotta-operator-controller-manager-5ffc74fd8c-wv9cm   2/2     Running   0          2m1s
+--->
+```
+
+Flotta is now running and ready to register edgedevices. To register a
+edgedevice we need first to retrieve the install scripts using the Makefile
+target `agent-install-scripts`.
+
+```
+make agent-install-scripts
+```
+
+Now, two scripts are created:
+  - hack/install-agent-dnf.sh: To install on normal Fedora installations
+  - hack/install-agent-rpm-ostree.sh: To install on rpm-ostree devices.
+
+Minikube is  a local virtual machine, so `$ minikube ip` cannot be reached from
```

Comment:
it depends on selected driver whether it will be a vm.

Reply:
nit: double space between "is  a "